worker/upload.go

```diff
@@ -198,7 +198,7 @@ func (w *worker) upload(ctx context.Context, r io.Reader, contracts []api.Contra
 			// upload packed slab
 			if len(packedSlabs) > 0 {
 				if err := w.tryUploadPackedSlab(ctx, mem, packedSlabs[0], up.rs, up.contractSet, lockingPriorityBlockedUpload); err != nil {
-					w.logger.Errorf("couldn't upload packed slabs, err: %v", err)
+					w.logger.Error(err)
```

Comment:
I'd leave the context here. Otherwise it might be hard to figure out where this is coming from.

Reply:
I don't mind bringing it back but did you see we call `.tryUploadPackedSlab` twice and (now) log the error without context twice? The reason for dropping it is because `tryUploadPackedSlab` returns errors with context and coincidentally the last context is identical to the one we had here.. so we were stuttering: `couldn't upload packed slab, err: couldn't upload packed slab, err: ...`